mlir/include/mlir/Dialect/Utils/StaticValueUtils.h

```diff
@@ -28,6 +28,10 @@ namespace mlir {
 /// with attribute with value `0`.
 bool isZeroIndex(OpFoldResult v);
 
+/// Return true if `v` is an IntegerAttr with value `1` of a ConstantIndexOp
+/// with attribute with value `1`.
+bool isOneIndex(OpFoldResult v);
```

Comment:
nit: Swap `Index` for `Integer`. This helper is looking for `IntegerAttr`.

Reply:
Do we use `Int` instead, which is shorter?